### DISCUSSION TITLE
Citations questions

### DISCUSSION THREAD

**Discussion by psmedia** [knowledgefutures/pubpub#3057] [Support]
Hi! I know citations are tricky, but we're working through things. However this latest file is throwing me for a loop. All our citations are in structured data, either bibtex or DOIs. All the citations show a "1. (and then a newline)" in front of them in the pop-up/modal. Down at the bottom of the webpage, the citation looks ok, but the same number is introduced in the citations in the pdf download, looking like:

1. 
1.
Health Emergency Risk Management [Internet]. 2019. Available from: https://apps.who.int/iris/bitstream/handle/10665/340893/WHO-EURO-2019-2378-42133-58035-eng.pdf
↩
2.
1.
Cinar P, Bold R, Bosslet BA, Bota DA, Burgess D, Chew HK, et al. Planning for post‐pandemic cancer care delivery: Recovery or opportunity for redesign? CA A Cancer J Clinicians [Internet]. 2020 Sep 30;71(1):34– 46. Available from: http://dx.doi.org/10.3322/caac.21644
↩ 

The docx download is also failing, for the same reason? 

Is this anything you've seen before? We are not getting this in other pubs. 
[f06lgvfbmhjtjdz76jawdmvuhoue1wut.pdf](https://github.com/pubpub/pubpub/files/14750196/f06lgvfbmhjtjdz76jawdmvuhoue1wut.pdf)

**Comment by gabestein** [comment]
I haven't seen this before, it's quite odd. Are you using Vancouver in other pubs as well?

**Reply by psmedia** [reply]
@gabestein - On checking, I see this that is a more systemic problem. We get the spurious 1. introduced into pop-ups for all citations using structured data. So this seems programmatic and a PubPub bug. Can you reproduce the issue?  

We do **not** get them in citations for which we use rich text. Our working plan was to put all citations in structured data, so that they exported to MS Word more effectively, but this is now in question. Hope we can deal with this as a bug?  

Other files show this issue, but they still download properly.

**Reply by gabestein** [reply]
As I asked above, can you please let me know if all those pubs are using Vancouver style, or if you're seeing it with other styles as well? I asked because I actually think it's an issue with the way the library we use is rendering Vancouver specifically. I'm not seeing it with other styles.

**Reply by psmedia** [reply]
Yes, we are only using Vancouver.

**Reply by psmedia** [reply]
Hi @gabestein - this one is still open for us. We are using Vancouver exclusively. If we include the citation as structured data, we get the weird 1. in the pop-up. If we include the citation as rich text, there's no weird number 1. in the pop-up, but the citations do not export in MS Word format (which we were hoping to use for workflow reasons...)

**Reply by gabestein** [reply]
Understood. We are trying to track it down, but as I wrote above, we believe it is a bug with the way the library we use to generate citations handles Vancouver, so a fix may not be under our control. I'll let you know when we have a sense of a fix and timeline.

**Reply by psmedia** [reply]
Do let me know what you find. I would like to have this pinned down.

**Reply by psmedia** [reply]
Colleagues, all of a sudden our structured data citations are failing, giving us an error. TIming out from the lookup? In general we're seeing an increase in application errors as well, not sure if that is connected to our heavy use of structured-data Vancouver citations.

**Reply by gabestein** [reply]
That service relies on Crossref, which is having ongoing issues with its public API: https://status.crossref.org/

Unfortunately, there's not a lot we can do until it's stabilized.

**Comment by Alexaandrew** [comment]
https://alexadigital.pubpub.org/pub/zgv5xgmx/release/1

**Comment by psmedia** [comment]
Understood! Thanks.

**Reply by gabestein** [reply]
A heads up that we've published a hotfix while we sort out the changes to Crossref's API. Pages will load much faster, and if a DOI lookup fails it will show the raw DOI rather than an error. We're working on longer-term approaches, but for now this should get you back to being able to work on those pubs, anyway.
